src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
1) Timer must close, because it will do some clean up jobs.

2) Must client.close first, then timer.close, because if timer is closed, and event loop group not close,  it may be possible to add new timeout task to timer, and timer will throw IllegalStateException.

3) Timer better not be daemon, if it is daemon, it may stop at any line of code,  first, HashedWheelTimer.Worker can't clean up, second, the retry task may be stopped, and the request will not get any response.